website/Contribution/CONTRIBUTING.rst

```diff
@@ -1160,8 +1160,7 @@ Please respect a few basic rules:
   `OCA Core Maintainers <https://github.com/orgs/OCA/teams/core-maintainers>`_.
   can do the job. You can notify them on Github using ``@OCA/core-maintainers``)
 * If you are in a hurry just send a mail at
-  contributors@odoo-community.org or ask by IRC (FreeNode
-  oca, openobject channel).
+  contributors@odoo-community.org or ask by chat (`OCA discord server <https://discord.gg/rN5xRdE>`_).
```

Comment:
It'd be nice to mention how it's bridged to Matrix community [+oca:matrix.org](https://matrix.to/#/+oca:matrix.org)

Reply:
How is it bridged to Matrix?

Reply:
Check the announcement in https://odoo-community.org/groups/contributors-15/contributors-183472?mode=thread&date_begin=&date_end=

Reply:
@Yajo What do you suggest to write? Thanks

Reply:
```suggestion
  contributors@odoo-community.org or ask by chat in either of:
  
  - `OCA discord server <https://discord.gg/rN5xRdE>`_
  - `Matrix community +oca:matrix.org <https://matrix.to/#/+oca:matrix.org>`__ (bridged to discord).
```

Reply:
Done, thanks